app/views/content/welcome/landing/_exciting-times-ahead.html.erb

```diff
@@ -31,6 +31,23 @@
     <div class="stage">
       <div class="card">
         <div class="number">2</div>
+        <%= image_pack_tag("media/images/content/welcome-guide/person-paper.svg", alt: "") %>
+        <div class="stage-info">
+          <h3 class="stage-name">Check your qualifications.</h3>
+          <p>
+            If you’ve got a degree or equivalent qualification, you’re ready
+            for a teacher training courses.
+          </p>
+          <a href="/funding-your-training">
```

Comment:
```suggestion
          <a href="/ways-to-train">
```

Reply:
I know Havas didn't supply a link but I think this is probably better going to Ways to train as that does talk about QTS, PGCE, etc.